scripts/swupd_performance/staging_file_gen.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+
+echo "OUTFILE=/var/log/swupd/staging/file_""$(date +%Y%m%d_%H%M%SZ)" > /etc/swupd_perform_update_config/script_config
```

Comment:
are you adding a "Z" at the end on purpose? `%H%M%SZ`

Reply:
Yes thats for timezone.

Reply:
in that case you are missing a %

Reply:
👍